website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/example-customizations.mdx

```diff
@@ -7,190 +7,154 @@ description: Example customizations for the ServiceNow Service Catalog Integrati
-      1. Click "Done" to finish configuring this Action.
-   1. Unfold the second Action in the Flow and click on the arrow to open it in
+      1. Click **Done** to finish configuring this Action.
+   1. Unfold the second Action in the Flow and click the arrow to open it in
       the Action Designer.
       ![Screenshot: ServiceNow Open Action Designer](/img/docs/servicenow-catalog-open-action.png "Screenshot of the ServiceNow Action Designer")
-      1. Click on the 3 dots in the top right corner and select "Copy Action".
+      1. Click **...** in the top right corner and select **Copy Action*".
          ![Screenshot: ServiceNow Copy Action](/img/docs/servicenow-catalog-copy-action.png "Screenshot of the ServiceNow Copy Action")
-         Rename it and click "Copy".
+         Rename it and click **Copy**.
          ![Screenshot: ServiceNow Rename Action](/img/docs/servicenow-catalog-rename-action.png "Screenshot of the ServiceNow Rename Action")
       1. In the the Inputs section, remove any previous example variables.
       ![Screenshot: ServiceNow Remove Variables From Action](/img/docs/servicenow-catalog-remove-example-variables-from-action.png "Screenshot of the ServiceNow Action Input Variables")
-      1. Add your custom variables by clicking the "Create Input" button. Ensure
-         that the variable names match your Catalog Item variables. Select the
-         appropriate type for each variable. Click "Save" at the top to save the
-         changes.
+      1. Add your custom variables by clicking the **Create Input** button. Ensure
+         that the variable names match your Catalog Item variables and select the
+         variable type that matches each variable. Click **Save**.
          ![Screenshot: ServiceNow Add Variables To Action](/img/docs/servicenow-catalog-add-variables-to-action.png "Screenshot of adding ServiceNow Action Input Variables")
-      1. Open the "Script step" within the Action. Remove any example variables
-         and add your custom variables by clicking "Create Variable" at the
+      1. Open the **Script step** within the Action. Remove any example variables
+         and add your custom variables by clicking **Create Variable** at the
          bottom. Enter the name of each variable and drag the corresponding data
-         pill from the right into the "Value field".
+         pill from the right into the **Value field**.
          ![Screenshot: ServiceNow Add Script Step Variables To Action](/img/docs/servicenow-catalog-adjust-script-variables.png "Screenshot of adjusting ServiceNow Action Script Variables")
-      1. Click "Save" and then "Publish" to save the changes made to the Action.
-1. Finally, reopen the Flow and attach the newly created Action to the Flow
+      1. Click **Save** and then **Publish**.
+1. Reopen the Flow and attach the newly created Action to the Flow
    after "Get Catalog Variables" step:
    1. Remove the previous Action that you copied earlier and replace it with
```

Comment:
This sentence is kind of confusing, I think the user will need to know exactly which action to copy and replace here with either an example name or a link to a previous step

Reply:
There's a screenshot visualizing this step below, if it helps :)

Reply:
We try not to rely on screenshots to bridge word gaps for folks to ensure we are prioritizing accessible and understandable language for those using screenreaders. 

@jastr945 I think this is referencing the second action, correct?

```suggestion
   1. Remove the second Action that you previously copied earlier and replace it with
```

Reply:
Let's reference it by name ("Create Terraform Workspace with Vars") - added a suggestion.